### DISCUSSION TITLE
Laika in Playwright

### DISCUSSION THREAD

**Discussion by ghost** [zendesk/laika#44] [Q&A]
Hi, 
I am interested into introducing Laika into tests written in Playwright. Do you have any experience with this library and suggest how to start using Laika with it?

**Comment by niieani** [comment]
Hi @vyrly! You should be able to use Laika similarly to the usage described for Cypress tests in any e2e testing solution, including Playwright or Puppeteer.

**Comment by svitlanaduman** [comment]
Hi I'm also very interested in this approach, I can't make it run. I'm using playwright component testing, They suggest to create a server to use beforeMount method in order to prepare everything for the test. But when I create in this method laika and intercepts it throws exceptions. I'm really got stucked too.
